src/GoogleFit/FitOnFhir.GoogleFit/Client/Telemetry/GoogleFitExceptionTelemetryProcessor.cs

```diff
@@ -4,18 +4,12 @@
 // -------------------------------------------------------------------------------------------------
 
 using EnsureThat;
-using Microsoft.Health.Common.Telemetry;
 using Microsoft.Health.Logging.Telemetry;
 
 namespace FitOnFhir.GoogleFit.Client.Telemetry
 {
     public class GoogleFitExceptionTelemetryProcessor : ExceptionTelemetryProcessor
     {
-        public GoogleFitExceptionTelemetryProcessor()
-            : base(typeof(AggregateException))
```

Comment:
By removing this, that means all exceptions passed in to this `ExceptionTelemetryProcessor` will be considered unhandled.  I'm not sure if that's the desired behavior you want or not.

Reply:
That is the behavior we want. We shouldn't be handling any exceptions here. All exception handling takes place in other places in the code, and this is just masking AggregateExceptions that could occur.